aiohttpretty.py

```diff
@@ -37,7 +49,9 @@ def fake_request(self, method, uri, **kwargs):
         yield from self.process_request(**kwargs)
         self.calls.append(self.make_call(method=method, uri=uri, **kwargs))
         mock_response = aiohttp.client.ClientResponse(method, uri)
-        mock_response._content = response.get('body', 'aiohttpretty')
+        content = response.get('body', 'aiohttpretty')
+        mock_response._content = content
+        mock_response.content = _MockStream(content)
```

Comment:
Could there be a check for if content is a stream? Such that content is a string bytes or stream.